src/App/Fossa/FossaAPIV1.hs

```diff
@@ -375,18 +375,18 @@ archiveUpload ::
   (Has (Lift IO) sig m, Has Diagnostics sig m) =>
   SignedURL ->
   FilePath ->
-  m ()
+  m String
```

Comment:
[nit] String is vague; we may want to pass out the LbsResponse and move the `show` to `compressAndUpload`